docs/versioned_docs/version-v28/02-guide/05-loan.md

```diff
@@ -384,6 +385,32 @@ func (k msgServer) LiquidateLoan(goCtx context.Context, msg *types.MsgLiquidateL
 }
 ```
 
+```go title="x/loan/keeper/msg_update_params.go"
+package keeper
+
+import (
+	"context"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"loan/x/loan/types"
+)
+
+func (k msgServer) UpdateParams(goCtx context.Context, req *types.MsgUpdateParams) (*types.MsgUpdateParamsResponse, error) {
+	if k.GetAuthority() != req.Authority {
+		return nil, errorsmod.Wrapf(types.ErrWrongLoanState, "invalid authority; expected %s, got %s", k.GetAuthority(), req.Authority)
```

Comment:
```suggestion
		return nil, errorsmod.Wrapf(types.ErrInvalidSigner, "invalid authority; expected %s, got %s", k.GetAuthority(), req.Authority)
```

Reply:
But we are missing [ErrInvalidSigner](https://github.com/ignite/cli/blob/main/docs/versioned_docs/version-v28/02-guide/05-loan.md?plain=1#L389-L400). Should we add this error type?

Reply:
Yes!

Reply:
done